conn_test.go

```diff
@@ -371,19 +371,6 @@ func TestResumption(t *testing.T) {
 	assertEquals(t, clientConfig.PSKs.Size(), 1)
 	assertEquals(t, serverConfig.PSKs.Size(), 1)
 
-	clientCache := clientConfig.PSKs.(*PSKMapCache)
-	serverCache := serverConfig.PSKs.(*PSKMapCache)
-
-	var serverPSK PreSharedKey
-	for _, key := range *serverCache {
-		serverPSK = key
-	}
-	var clientPSK PreSharedKey
-	for _, key := range *clientCache {
-		clientPSK = key
-	}
-	assertDeepEquals(t, clientPSK, serverPSK)
```

Comment:
Even though the PSKs won't match exactly, we should check that the parts that should be the same are the same.  Could you please at least check that the identity and key are the same?